package.json

```diff
@@ -1,27 +1,27 @@
 {
   "name": "registry-viewer",
-  "author": "Paul Schultz & Nadia Waid",
+  "author": "Red Hat",
   "version": "1.0.0",
   "scripts": {
     "dev": "next",
     "build": "next build && next export",
     "start": "next start",
-    "start-build": "yarn build & yarn start",
     "clean": "rm -rf .next/ out/ node_modules/",
-    "cypress:start": "yarn cypress open",
-    "cypress:start-build": "concurrently --names 'CYPRESS,SERVER' --prefix-colors 'yellow,blue' \"yarn cypress:start\" \"yarn start-build\"",
+    "cypress:start": "concurrently --names 'CYPRESS,SERVER' --prefix-colors 'yellow,blue' \"yarn cypress open\" \"yarn build && yarn start\"",
     "typedoc:build": "node_modules/.bin/typedoc --tsconfig .",
     "typedoc:start": "npx serve docs",
     "jest:test": "jest --watchAll --verbose",
     "test": "cypress run",
     "lint": "prettier --check .",
-    "format": "prettier --write ."
+    "format": "prettier --write .",
+    "prepare": "husky install"
```

Comment:
How does husky pre commit work?

I think it would be nice if we can have a git workflow that does this check for every PR opened and fails if things are not pretty. 

Like this for example - https://github.com/devfile/api/blob/main/.github/workflows/ci.yaml#L55-L58

Reply:
Yeah I was kinda thinking this as well. I just havent looked into it yet.

Reply:
Added. husky pre commit triggers a format before commiting.